packages/base-driver/lib/basedriver/driver.ts

```diff
@@ -161,6 +161,9 @@ export class BaseDriver<
 
     // log timing information about this command
     const endTime = Date.now();
+
+    cmd = this.modifyCommandName(cmd, args);
```

Comment:
I was thinking about the implementation and it looks like there is a way to retrieve a name of an execute extension. You may check the static `executeMethodMap`  property of the driver. It should contain commands mapping as in https://github.com/appium/appium-xcuitest-driver/blob/master/lib/execute-method-map.ts
I assume we could use the `command` property there to fetch the actual name

Reply:
Thank you for that. That sounds good. Will keep in mind when making changes in the xcuitest-driver once these changes are merged.

Reply:
Ahh I understand better now. Apologies. Thank you so much for clarifying that. I will revisit this PR with the approach suggested.